test_data/yast/autoyast_reinstall/autoyast_reinstall_64bit.yaml

```diff
@@ -4,9 +4,5 @@ paths:
   - /var/lib/polkit/.bashrc
   - /var/lib/nobody/.bashrc
   - /var/lib/pulseaudio/.bashrc
-device: vda
-table_type: gpt
-file_system:
-  /: btrfs
-swap: 1
+<<: !include test_data/yast/autoyast_reinstall/partitioning_no_home.yaml
 <<: !include test_data/yast/autoyast/profiles/reinstall.yaml
```

Comment:
Maybe here?

Reply:
I notice for autoyast mini the same, when I have two includes it doesn't work, do you know why? maybe I just missing something trivial...

Reply:
I found the issue in the script, see [commit](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12467/commits/423bd035ac181d5c696d1d9bd6faf1b6671de608#diff-373ab693c2f2beb001873bf44403dfe9b3cb9def1c54fb774cd80049bbae6752R26)